### PR TITLE
Add IBM System Storage Block CSI driver version 1.10.0 files

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/metadata.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "1.10.0",
+  "revision": "2",
```
Comment (mssachan) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/metadata.json:3]: Change `revision` to `1`

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/metadata.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      "name": "support-url",
+      "value": "https://www.ibm.com/support/home/"
+    },
+    {
+      "name": "support-msg",
+      "value": "For support contact IBM"
+    },
+    {
+      "name": "provider-logo",
+      "value": "logo/ibm-system-storage-block-csi-driver.png"
```
Comment (mssachan) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/metadata.json:38]: Change path for svg image
`logo/ibm-system-storage-block-csi-driver.svg`

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-csi-block
+  namespace: kube-system
+  annotations:
+    version: 1.10.0
+    revision: 2
```
Comment (mssachan) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/deployment.yaml:8]: Change `revision` to `1`

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+| `prefix` | Optional | The prefix name of the volume that is created. | N/A |
+| `VolumeExpansion` | Optional |  Specify `true` to allow volume expansion or `false` to disallow volume expansion. | `false` |
+
+
+## Creating the IBM block storage CSI driver storage configuration
+
+**Example `sat storage config create` command**
+
+```sh
+ibmcloud sat storage config create --name <config-name> --template-name ibm-system-storage-block-csi-driver --template-version 1.10.0 -p "namespace=<namespace>" -p "name=<sc-name>" -p "space-efficiency=<space-efficiency>" -p "pool=<pool>" -p "secret-name=<secret-name>" -p "secret-namespace=<secret-namespace>" -p "fstype=<fstype>" -p "prefix=<prefix>" -p "VolumeExpansion=<VolumeExpansion>"
```
Comment (gauravAggrahari) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md:46]: --location flag is missing.

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+```
+
+## Creating the storage assignment
+
+**Example `sat storage assignment create` command**
+
+```sh
+ibmcloud sat storage assignment create --name <assignment-name> --group <cluster-group-name> --config <config-name>
```
Comment (gauravAggrahari) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md:55]: We need add the command for `cluster` as well

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+## Verifying your IBM block storage CSI driver storage configuration is assigned to your clusters
+
+Run the following commands to verify that the driver pods are running.
+
+```bash
+$ kubectl get all -n <namespace>  -l csi
```
Comment (gauravAggrahari) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md:63]: Also check for storage classes creation on the cluster

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+statefulset.apps/ibm-block-csi-controller   1/1     9m36s
+```
+## Removing your assignment
+
+When you remove the storage assignment from your clusters, the following resources are removed.
+ - All pods, daemonsets and statefulsets from the deployment.
+ - All objects that are created manually will remain on the cluster. For example: app pods, PVC, PVs that you created manually, must be manually removed.
+  
+```sh
+ibmcloud sat storage assignment rm --assignment <assignment-name>
```
Comment (gauravAggrahari) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/README.md:82]: assignment-name --> assignment-id

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/metadata.json (hunk 2 above)
Comment (gauravAggrahari) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/metadata.json:38]: change it .svg

File: config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/storage-class-parameters.json [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+    "templateKeyName": [
+      "storageclass.kubernetes.io/is-default-class"
+    ],
+    "default": "false",
+    "required": "false",
+    "category": "config",
+    "type": "boolean"
+  }, {
+    "description": "Specify a unique volume tier preference; in the range of 1-99 with 1 being the highest",
+    "displayname": "Storage Class Preference Preference",
```
Comment (mssachan) [inline config-templates/ibm/ibm-system-storage-block-csi-driver/1.10.0/storage-class-parameters.json:99]: `Storage Class Preference Preference` --> `Storage Class Preference`

**Review by mssachan — APPROVED** [review]

**Pull request merged**